gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigsResource.java

```diff
@@ -79,7 +89,33 @@ public FlowConfig get(ComplexResourceKey<FlowId, EmptyRecord> key) {
    */
   @Override
   public CreateResponse create(FlowConfig flowConfig) {
-    return this.getFlowConfigResourceHandler().createFlowConfig(flowConfig);
+    ExecutionStatus latestFlowExecutionStatus = getLatestExecutionStatus(flowConfig);
+    if (latestFlowExecutionStatus == ExecutionStatus.RUNNING) {
```

Comment:
Isn't this check not reliable since the flow status is not immediately available upon job creation? Why not check against the flow catalog?